index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Newest Questions - Stack overflow </title>
+    <link rel="stylesheet" href="style.css">
+    <link href="utils.css" rel="stylesheet">
+</head>
+
+<body>
+    <nav>
+        <a href="https://stackoverflow.com/">
+            <img src="./assets/1280px-Stack_Overflow_logo.svg.png" alt="logo" width="150"> </a>
+        <a href="https://stackoverflow.co/"> About </a>
+        <p> Products</p>
+        <a href="https://stackoverflow.co/teams/"> For teams </a>
+        <form>
+            <input placeholder=" Search..." type="text" name="search">
+            <svg class="search-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                <!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                <path
+                    d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352c79.5 0 144-64.5 144-144s-64.5-144-144-144S64 128.5 64 208s64.5 144 144 144z" />
+            </svg>
```

Comment:
The form should have a `button` with `type="submit"`. This way, if you click ENTER, the form/search will be performed.

We need to hide the buton, so you can do that with `hidden` attribute.